homeassistant/components/luftdaten/sensor.py

```diff
@@ -30,7 +30,12 @@ async def async_setup_entry(hass, entry, async_add_entities):
 
     sensors = []
     for sensor_type in luftdaten.sensor_conditions:
-        name, icon, unit = SENSORS[sensor_type]
+        try:
+            name, icon, unit = SENSORS[sensor_type]
+        except KeyError:
+            _LOGGER.debug("Unknown sensor value type: %s", sensor_type)
+            continue
```

Comment:
I believe this `continue` statement is not needed here :)
```suggestion
```

Reply:
It is. These should be skip the append below.

Reply:
Oh right 🤦

Reply:
As @elupus said, the `continue` statement is needed, because in case of a `KeyError`, the `sensors.append()` function call in line 39 should not be executed.
But just saw, that this PR is already merged. Thanks.